docs/getting-started/delete-on-dockstore.rst

```diff
@@ -0,0 +1,44 @@
+Deleting an entry on Dockstore
+===================================
+
+Tutorial Goals
+--------------
+
+-  Deleting an entry
+-  Uninstalling the Dockstore GitHub App from the source GitHub repo
+
+This tutorial walks through the process of deleting entries from Dockstore.
+This action is permanent and after you delete the workflow, it will no longer appear on Dockstore.
```

Comment:
Nit: You're following the dialog text, where I should have noticed it earlier, but it seems little dissonant to on the one hand say it's permanent and will no longer appear on Dockstore, and then there is a section below on how to ensure is doesn't appear again -- that's not permanent. :)

Not sure I have an easy rephrasing in mind, so just something to ponder, and if no other idea comes, ok to leave.

Reply:
Kept line to keep consistent with dialog